ui/cypress.json

```diff
@@ -1,4 +1,5 @@
 {
   "baseUrl": "http://localhost:3000",
-  "experimentalNetworkStubbing": true
+  "experimentalNetworkStubbing": true,
+  "video": false
```

Comment:
Let's have it to true, we can see the test video. Moreover, we can apply for cypress dashboard too as we are open-source 😎

Reply:
Great! Have the Cypress Dashboard, sounds really good